### ISSUE TITLE
[BUG] Visual boundary of form fields lacks 3 to 1 color contrast

### ISSUE THREAD

**Issue by mndonx** [emulsify-ds/compound#75]
**Describe the bug**
Visual boundary of form fields lacks 3 to 1 color contrast.

**To Reproduce**
1. Using Chrome, Open https://emulsify-ds.github.io/compound/?path=/story/atoms-forms--textfields-examples
2. Now measure the color contrast for all the text field boundaries against the background white color
3. Currently, the visual boundary (light grey border) of the all the text fields lacks 3 to 1 color contrast

Foreground: #E5E5E5
Background: #FFFFFF
The contrast ratio is: 1.3:1

**Expected behavior**
Non text elements like boundaries, focus indicators, icons must have at least 3 to 1 color contrast against the background

People who have low vision or are colorblind may have difficulty perceiving the full hit area of the element

Ensure that boundaries have at least 3 to 1 color contrast against the background. If the boundary  color is changed to #949494. New contrast ratio: 3: 1

Refer:
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast
https://accessible-colors.com/

**Screenshots**
![Visual boundary of form fields lacks 3 to 1 color contrast](https://user-images.githubusercontent.com/2343957/208309479-12be8b0a-bbbb-4612-ab5f-1c21035ef5cc.png)

**Desktop (please complete the following information):**
 -All

**Smartphone (please complete the following information):**
 - All
